scripts/install-relayer.sh

```diff
@@ -2,37 +2,39 @@
 
 RESET_CONFIG="${RESET_CONFIG:-0}"
 VERSION="${VERSION:-v2.0.0}"
-RELAYER_PATH=$HOME/.relayer
-RELAYER_CONFIG=$RELAYER_PATH/config/config.yaml
+RELAYER_HOME="${RELAYER_HOME:-$HOME/.relayer}"
+RELAY_CMD="${RELAY_CMD:-rly}"
+RELAYER_CONFIG="$RELAYER_HOME"/config/config.yaml
```

Comment:
Usually you would want the double quote on the end of this line not in the middle.

Reply:
Agreed, I'll make this change.